docs/source/perf/fix_aten_ops.md

```diff
@@ -0,0 +1,71 @@
+# Fix attention ops
+
```

Comment:
where is this doc coming from? It felt like a prt of the export but does not have much context and is confusing.

Reply:
This content came from xla/docs/FIX_LOWERING_FOR_CORE_ATEN_OPS.md

Since it doesn't have much content or context, I will remove it.

Reply:
you can just add it back to the `FIX_LOWERING_FOR_CORE_ATEN_OPS ` I guess

Reply:
The file has been renamed to fix_aten_ops.md. I can re-add it, but if it is confusing, without context, and without a specific purpose, I'd rather leave it out.